changelog.rst

```diff
@@ -6,7 +6,9 @@ Release History
 0.6.5 (dev)
 -----------
 
-|new| New model-wide constraints `supply_min` and `supply_max` which restrict the absolute energy produced by a subgroup of technologies and locations.
+|new| New model-wide constraints `energy_cap_equals`, `resource_area_equals`, and  `energy_cap_share_equals` to add the equality constraint to existing `min/max` group constraints.
+
+|new| New model-wide constraints `carrier_prod_min`, `carrier_prod_max`, and  `carrier_prod_equals` which restrict the absolute energy produced by a subgroup of technologies and locations.
```

Comment:
We should at least mention that this is part new and part renaming of existing constraints

Reply:
I think this is all new, as far 0.6.5 is concerned. They were introduced as `supply_max/min` in 0.6.5-dev, so renaming them doesn't really matter for the changelog, so long as the name is fixed by the next stable release?